mobile-widgets/themeinterface.cpp

```diff
@@ -26,6 +26,7 @@ void themeInterface::setup(QQmlContext *ct)
 	instance()->set_currentScale(qPrefDisplay::mobile_scale());
 }
 
+
```

Comment:
that's an interesting commit. I think my small cleanup caused your tooling to add this...
obviously this commit should be dropped